test/functional/test-ad-cid.js

```diff
@@ -73,7 +73,7 @@ function tests(name) {
         });
 
         it('provides cid to ad', () => {
-          adConfig['_ping_'] = {clientIdScope: cidScope};
+          config.clientIdScope = cidScope;
```

Comment:
I set the cidScope explicitly in each test. It's a better way to "document" the test.

Of course you can still argue we should still reset, to remove the side effect and avoid traps for future test, though here it makes no difference to current tests.